src/docs/development/platform-integration/platform-channels.md

```diff
@@ -44,6 +44,12 @@ channels as illustrated in this diagram:
 Messages and responses are passed asynchronously,
 to ensure the user interface remains responsive.
 
+{{site.alert.note}} 
+  Even though Flutter sends messages to and from Dart asynchronously,
+  whenever you invoke a channel method, you must invoke that method on the
+  platform's main thread.
+{{site.alert.end}}
```

Comment:
Maybe some examples of how to do that, or a link to the place with examples?

Reply:
It may also be worth explaining what "main thread" really means in relationship to the different platforms. The term is a little overloaded.

Reply:
@dnfield @mklim good suggestions. I pushed an update that links to official docs about each platform's main thread, and I added examples in each language. Please let me know if it looks good.